resources/functions/tenant-registrations/index.py

```diff
@@ -194,10 +223,27 @@ def delete_tenant_registration(tenant_registration_id: str):
             offboarding_detail_type,
         )
 
-        return {"message": "Tenant registration deletion initiated"}
+        return {
+            "data": {"message": "Tenant registration deletion initiated"}
+        }, HTTPStatus.OK
     except ClientError as e:
         logger.error(f"Error deleting tenant registration: {str(e)}")
-        return {"message": "Internal server error"}, HTTPStatus.INTERNAL_SERVER_ERROR
+        raise InternalServerError("Unknown error during processing!")
+
+
+def __update_tenant_registration(tenant_registration_id, update_data):
```

Comment:
At some point (not right now), we should put a `created` and `modified` date in all of these `tenant-*` tables and set appropriately. I'll put a gh issue in to discuss